SConstruct

```diff
@@ -2319,6 +2319,7 @@ if env["WITH_GL"] and doConfigure :
 
 		glTest = glTestEnv.Command( "test/IECoreGL/results.txt", glPythonModule, "$PYTHON $TEST_GL_SCRIPT --verbose" )
 		NoCache( glTest )
+		glTestEnv.Depends( glTest, glPythonModule )
```

Comment:
I think this dependency is declared at L2320 - was that not working?

Reply:
Oh, I had noticed that it wasn't working, but I guess when I added glPythonModule, it ran just because the SConstruct was flagged as modified.  What was actually missing was glLibrary, I've switched it properly now.